src/Sarif.Viewer.VisualStudio/ErrorList/SarifTableDataSource.cs

```diff
@@ -29,20 +29,20 @@ class SarifTableDataSource : ITableDataSource
         private SarifTableDataSource()
         {           
             var compositionService = ServiceProvider.GlobalProvider.GetService(typeof(SComponentModel)) as IComponentModel;
-            compositionService.DefaultCompositionService.SatisfyImportsOnce(this);
+            compositionService?.DefaultCompositionService.SatisfyImportsOnce(this);
```

Comment:
>compositionService [](start = 12, length = 18)

Rather than repeatedly testing the result, how about:
```
// The composition service will only be null in unit tests.
if (compositionService != null)
{
    // Do everything else.
}
```